cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-release-information.adoc

```diff
@@ -9,9 +9,8 @@ On Prisma Cloud Enterprise Edition, the *Compute* tab comprises the Cloud Worklo
 
 // This release only includes fixes that are listed in xref:prisma-cloud-compute-known-issues.adoc#id311f617e-5159-4b1b-8cfa-29183c6e4a74[Prisma Cloud Compute Known Issues].
 The current *Compute* version is 30.02.123.
-//It will be upgraded to 30.03.xxx on Jul 16, 2023.
-To see the new capabilities and changes planned in July 2023, see the xref:look-ahead-planned-updates-prisma-cloud-compute.adoc[Prisma Cloud Compute Look Ahead].
-
+It will be upgraded to 30.03.xxx on Jul 16, 2023.
```

Comment:
Build number to be updated here, post the QA exit letter.

Reply:
Done